iced_layershell/Cargo.toml

```diff
@@ -11,14 +11,15 @@ description = "layershell binding for iced"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = ["tiny-skia", "wgpu", "fira-sans", "auto-detect-theme"]
+default = ["tiny-skia", "wgpu", "fira-sans", "auto-detect-theme", "tokio"]
```

Comment:
I think we should not modify it here, it should follow the iced

Reply:
It looks a little weird that it will fail to compile with default features.

If you think that is the expected behavior, I will remove it.

Reply:
ok, please remove it , the runtime should be decided by iced, not our part